hollow/src/main/java/com/netflix/hollow/tools/stringifier/HollowStringifier.java

```diff
@@ -44,6 +44,15 @@
      */
     public void stringify(Writer writer, HollowRecord record) throws IOException;
 
+    /**
+     * Writes a String representation of the specified collection of {@link HollowRecord} to the provided Writer.
+     *
+     * @throws IOException thrown if there is an error writing to the Writer
+     */
+    public default void stringify(Writer writer,Iterable<HollowRecord> records) throws IOException {
+        //Do nothing
```

Comment:
Maybe `throw new UnsupportedOperationException("not implemented")`? My gut tells me that's a default that would surface errors sooner.

Reply:
hmmm... actually, I think it makes sense to throw the exception. That gives more visibility and could prevent silent errors in the future. Changed in https://github.com/Netflix/hollow/pull/245/commits/14a501eb7e9ecc99f522064b7f76972958afcb8b